README.md

```diff
@@ -16,8 +16,16 @@ http://opensource.addepar.com/ember-charts/
 
 http://emberjs.jsbin.com/rekawobugu/1/edit
 
+Unfortunately, this version of Ember Charts is out of date,
+but the current maintainers of Ember Charts at Addepar have not been
```

Comment:
I changed it to and. Let's own our mistakes, or some other similarly inspirational phrase.